src/resolvers/Mutation.ts

```
@@ -0,0 +1,25 @@
+const resolvers = {
+	Mutation: {
+		addOrganizationCustomField: async (
+			_: any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)